README.md

```diff
@@ -63,6 +63,27 @@ client.payee_details('fake-payee-id').body
     "AccountNumber"=>"123456789",
     "RoutingNumber"=>"121042882",
     "AccountType"=>"S"}}
+    
+```
+
+##### Performing a payout with expanded params:
```

Comment:
Can we also add the case when we would be sending a payout with expanded params?